app/emails/bookings-updates-template.tsx

```diff
@@ -0,0 +1,116 @@
+import type { Booking } from "@prisma/client";
+import {
+  Button,
+  Html,
+  Text,
+  Img,
+  Link,
+  Head,
+  render,
+  Container,
+  Heading,
+} from "@react-email/components";
+import { SERVER_URL } from "~/utils/env";
+import { styles } from "./styles";
+
+interface Props {
+  heading: string;
+  booking: any;
+  assetCount: number;
+}
+
+export function BookingUpdatesEmailTemplate({
+  booking,
+  heading,
+  assetCount,
+}: Props) {
+  return (
+    <Html>
+      <Head>
+        <title>Bookings update from Shelf.nu</title>
+      </Head>
+
+      <Container style={{ padding: "32px", textAlign: "center" }}>
+        <Img
+          src="cid:shelf-logo"
+          alt="Shelf's logo"
+          width="100"
+          height="32"
+          style={{ margin: "0 auto 32px auto" }}
+        />
+        <div style={{ marginBottom: "32px" }}>
+          <Heading as="h1" style={{ ...styles.h1 }}>
+            {heading}
+          </Heading>
+          <Heading as="h2" style={{ ...styles.h2 }}>
+            {booking.id} | {booking.name} | {assetCount}{" "}
+            {assetCount === 1 ? "asset" : "assets"}
+          </Heading>
+          <p style={{ ...styles.p }}>
+            <span style={{ color: "#101828", fontWeight: "600" }}>
+              Custodian:
+            </span>{" "}
+            {`${booking.custodianUser?.firstName} ${booking.custodianUser?.lastName}` ||
+              booking.custodianTeamMember?.name}
+          </p>
+          <p style={{ ...styles.p }}>
+            <span style={{ color: "#101828", fontWeight: "600" }}>From:</span>{" "}
+            {booking.from.toLocaleString()}
```

Comment:
That is not the correct way to format the date as it will not always use the correct style and formatting. Please check `baseBookingEmailContent` for an example how we format dates.